api-report/map.api.md

```diff
@@ -40,6 +40,7 @@ export class DirectoryFactory {
 // @public
 export interface IDirectory extends Map<string, any>, IEventProvider<IDirectoryEvents>, Partial<IDisposable> {
     readonly absolutePath: string;
+    countSubDirectory?(): number;
```

Comment:
name seems a bit weird, but I guess it's in sync with other methods, so that's ok.

Reply:
The size method was already there to count the number of keys. So this sound right to me as per the other sub directory related function names.